AGGREGATE.md

```diff
@@ -504,6 +504,9 @@ As the trigger context ID in the aggregatable report explicitly reveals the
 association between the report and the trigger, these reports can be sent
 immediately without delay.
 
+When a trigger context ID is provided, the aggregatable report will not count
+towards the limit of aggregatable reports per source, nor be limited by it.
```

Comment:
Is there a link to this limit? and might be worth rephrasing nor limited by it is slightly confusing technically its bounded by L1 limit?